include/openssl/ssl.h

```diff
@@ -338,6 +338,9 @@ typedef int (*SSL_verify_cb)(int preverify_ok, X509_STORE_CTX *x509_ctx);
 # define SSL_OP_ALLOW_UNSAFE_LEGACY_RENEGOTIATION        0x00040000U
 /* Disable encrypt-then-mac */
 # define SSL_OP_NO_ENCRYPT_THEN_MAC                      0x00080000U
+/* Prioritize Chacha20Poly1305 when client does.
+ * Modifies SSL_OP_CIPHER_SERVER_PREFERENCE */
+# define SSL_OP_PRIORITIZE_CHACHA                        0x00200000U
```

Comment:
What's rationale behind choice of constant? Basically there is contradiction. If rationale is that 0x00100000 was removed in 1.1.0, while 0x00200000 in 1.0.2, then it doesn't add up with previous constant. I mean if this is the reason that one probably should adjust even previous constant... But formally speaking removed constant should not be a problem, because you are forced to recompile when upgrading to 1.1.x and it's sufficient to just redefine corresponding macros as zeros. I mean at source code level they won't cause backward compatibility clash...

Reply:
The rational is that I have another outstanding PR #3910 using 0x00100000

Reply:
(There was a prior issue with values that were in SSL_OP_ALL in 1.1.0 and unused, but those values have been marked reserved until 1.2.0)